### PR TITLE
fix(macros/Deprecated_Header): add note msg when version is not provided

### PR DESCRIPTION
**Pull request by yin1999**
## Summary

The `Deprecated` note may be removed by mistake, just add it back when the version is not provided.

### Problem

See:

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menuitem

image

### Solution

When the version is not provided, use `Deprecated` note instead (copying from:

https://github.com/mdn/yari/blob/cd6c1176827a3afc8f53460c66280cd26db5804a/kumascript/macros/Deprecated_Inline.ejs#L18-L27

---

## Screenshots

See those pages:

- with version param: `http://localhost:5042/fr/docs/Web/HTML/Element/menuitem`

    image

- with version param (digits): `http://localhost:5042/es/docs/Web/API/IDBCursor#constants`

    image

- with version param (empty string): `http://localhost:5042/ru/docs/Web/JavaScript/Reference/Global_Objects/Date/getYear`

    image

- without version param: `http://localhost:5042/en-US/docs/Web/HTML/Element/menuitem`

    image

---

## How did you test this change?

run `yarn dev` and see the rendered pages.

### CODE REVIEW

File: kumascript/macros/Deprecated_Header.ejs [hunk 1]
```diff
@@ -3,12 +3,23 @@
 /* prepend version with HTML, CSS, JS or Gecko for those techs */
 /* for backwards compatibility unprefixed is Gecko, please avoid this */
 
-var str = $0;
+let version = $0;
 if (/^\d/.test($0)) {
-  str = "gecko" + $0;
+  version = `gecko ${version}`;
```
Comment (yin1999) [inline kumascript/macros/Deprecated_Header.ejs:8]: adding a space between `gecko` and the version. I think it's better

File: kumascript/macros/Deprecated_Header.ejs [hunk 2]
```diff
@@ -18,9 +29,10 @@ var str_desc = mdn.localString({
   "zh-TW": "不推薦使用此功能。雖可能有一些瀏覽器仍然支援它，但也許已自相關的網頁標準中移除、正準備移除、或僅為了維持相容性而保留。避免使用此功能，盡可能更新現有程式；請參考頁面底部的<a href='#瀏覽器相容性'>相容性表格</a>來下決定。請注意：本功能可能隨時停止運作。"
 });
 
+const noteMsg = version || note;
```
Comment (caugner) [inline kumascript/macros/Deprecated_Header.ejs:32]: Let's inline this variable in line 36.

```suggestion
```

**Review by caugner — CHANGES_REQUESTED** [review]
Good catch, thank you!

Just one nit.

File: kumascript/macros/Deprecated_Header.ejs [hunk 3]
```diff
@@ -18,9 +29,10 @@ var str_desc = mdn.localString({
...
 %>
 
 <div class="notecard deprecated">
-    <h4><%-str%></h4>
-    <p><%-str_desc%></p>
+    <h4><%-noteMsg%></h4>
```
[the remark this answers is not in the document]
Comment (yin1999) [reply kumascript/macros/Deprecated_Header.ejs]: Just made a local test and it works.

**Pull request merged**